client/payment-details/timeline/map-events.js

```diff
@@ -311,6 +311,25 @@ const composeFXString = ( event ) => {
 	);
 };
 
+// Conditionally adds the ARN details to the timeline in case they're available.
+const getRefundTrackingDetails = ( event ) => {
+	return 'available' === event.acquirer_reference_number_status
+		? `Acquirer Reference Number(ARN) ${ event.acquirer_reference_number }`
```

Comment:
This text should use `__()` for the localization purpose. 

Also nitpick: there may be a space before `(ARN)`.

Reply:
Fixed